.vib/owncloud/cypress/cypress/integration/owncloud_spec.js

```diff
@@ -4,77 +4,18 @@ import { random } from '../support/utils';
 it('allows creating a folder and uploading a file ', () => {
   cy.login();
   cy.get('.new').click();
-  cy.get('[data-action="folder"]').click();
+  cy.contains('Folder').click();
   cy.fixture('folders').then((folder) => {
     cy.get('[value="New folder"]').type(`${folder.newFolder.name}.${random}`);
     cy.contains('Create').click();
-    cy.contains('.innernametext', `${folder.newFolder.name}.${random}`).click();
+    cy.contains(`${folder.newFolder.name}.${random}`).click();
   });
   cy.get('.new').click();
   cy.get('[data-action="upload"]').click();
   cy.get('input[type="file"]').selectFile(
     'cypress/fixtures/file_to_upload.json',
     { force: true }
   );
-  cy.contains('No files in here').should('not.be.visible');
+  cy.contains('file_to_upload').should('be.visible');
```

Comment:
Would it be possible to expand this test so Cypress downloads the uploaded file? Given owncloud's file-sharing nature, we are only testing "half" of the use case here.